src/Adapter/Email/EmailConfigurationTester.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Adapter\Email;
+
+use PrestaShop\PrestaShop\Adapter\Entity\Mail;
+use PrestaShop\PrestaShop\Adapter\Entity\Tools;
+use PrestaShop\PrestaShop\Core\ConfigurationInterface;
+use PrestaShop\PrestaShop\Core\Email\EmailConfigurationTesterInterface;
+use PrestaShop\PrestaShop\Core\Email\MailMethodOption;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * Class EmailConfigurationTester is responsible for sending test email
+ */
+final class EmailConfigurationTester implements EmailConfigurationTesterInterface
+{
+    /**
+     * @var ConfigurationInterface
+     */
+    private $configuration;
+
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    /**
+     * @param ConfigurationInterface $configuration
+     * @param TranslatorInterface $translator
+     */
+    public function __construct(
+        ConfigurationInterface $configuration,
+        TranslatorInterface $translator
+    ) {
+        $this->configuration = $configuration;
+        $this->translator = $translator;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function testConfiguration(array $config)
+    {
+        $content = $this->translator->trans('This is a test message. Your server is now configured to send email.', [], 'Admin.Advparameters.Feature');
```

Comment:
Line too long, better to indent:
```php
$content = $this->translator->trans(
    'This is a test message. Your server is now configured to send email.', 
    [], 
    'Admin.Advparameters.Feature'
);
```

Easier to review and for other people that are not using PHPStorm :D